src/main/java/uk/gov/hmcts/darts/audio/service/impl/AudioTransformationServiceImpl.java

```diff
@@ -223,7 +225,15 @@ private void processAudioRequest(MediaRequestEntity mediaRequestEntity) {
     List<MediaEntity> filterMediaByMediaRequestTimeframeAndSortByStartTimeAndChannel(List<MediaEntity> mediaEntitiesForRequest,
                                                                                      MediaRequestEntity mediaRequestEntity) {
         return mediaEntitiesForRequest.stream()
+            .filter(media -> nonNull(media.getStart()))
+            .filter(media -> nonNull(media.getEnd()))
+            // Filter out media where the media start and media end times are the same and not less than a second apart
+            .filter(media -> !media.getStart().truncatedTo(ChronoUnit.SECONDS).isEqual(media.getEnd().truncatedTo(ChronoUnit.SECONDS)))
+            // Filter out media where the media start time is after the media end time
+            .filter(media -> media.getStart().isBefore(media.getEnd()))
```

Comment:
Is this filter and the one below valid use cases? What AC on the ticket do these relate to?